### PR TITLE
Clean up global commands and help text

### PR DESCRIPTION
**Pull request by gschier**
Changes:

- Renamed `disconnect` to `unlink` and split `init` into `init` and `link`
- Renamed `env` to `variables` to align verbiage with Dashboard
- Adjusted help text for all commands
- Both Stripe and Heroku CLIs were used as a style reference

image

Old renamed command still work but are deprecated.

image

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -52,61 +52,73 @@ func init() {
...
-		Short: "Disconnect from Railway",
+		Use:        "disconnect",
+		Short:      "Disconnect from Railway",
+		RunE:       contextualize(handler.Disconnect, handler.Panic),
+		Deprecated: "The 'railway disconnect' command is now railway 'disassociate'", /**/
+	})
+	rootCmd.AddCommand(&cobra.Command{
+		Use:   "disassociate",
+		Short: "Remove association between project and current directory",
 		RunE:  contextualize(handler.Disconnect, handler.Panic),
```
Comment (coffee-cup) [inline main.go:86]: suggestion: rename the handler function to `Disassociate`

**Review by coffee-cup — APPROVED** [review]

**Comment by JakeCooper** [comment]
I'm unsure how I feel about `init` and `disassociate` being complementary commands. They don't sound related and IDK how I feel about the verbage of `disassociate`

To me, it almost feels like they should be `connect` and `disconnect` and we could consider renaming `connect` to something else

Thoughts?

**Comment by gschier** [comment]
@JakeCooper ya, it still seems a bit weird to me too. I was also thinking that `disassociate` may be an infrequent-enough command to put under `railway init remove` or something.

**Comment by gschier** [comment]
We could rename `init` to `link/unlink` but that doesn't imply the ability to create a project as much as `init` does. Or we could rename `disassociate` to `unlink` (didn't think of that verb before).

**Pull request merged**